pkgs/development/python-modules/aiolifx-themes/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, aiolifx
+, poetry-core
+, pytest
+, pytest-cov
+, pytest-asyncio
+, pytestCheckHook
+, async-timeout
+, typer
+}:
+
+buildPythonPackage rec {
+  pname = "aiolifx-themes";
+  version = "0.2.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "Djelibeybi";
+    repo = "aiolifx-themes";
+    rev = "v${version}";
+    hash = "sha256:17498vdg8i20hk4i8hzc67qaj206ik3s1zn1k70plsjr9zlgs6vz";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiolifx
+    typer
+  ] ++ typer.optional-dependencies.all;
+
+  checkInputs = [
+    async-timeout
+    pytest
+    pytest-cov
```

Comment:
Let's substitute coverage away, we gain no benefit by running it.

https://github.com/Djelibeybi/aiolifx-themes/blob/main/pyproject.toml#L57